lib/stripe/stripe_object.rb

```diff
@@ -262,9 +262,10 @@ def method_missing(name, *args)
       # TODO: only allow setting in updateable classes.
       if name.to_s.end_with?('=')
         attr = name.to_s[0...-1].to_sym
+        val = args.first
```

Comment:
Yeah ... I don't think we can always rely on a value always being available, but on the other hand, I think this code is safe enough by default that it doesn't matter that much. Worst case we get a `nil` back and that gets ignored by `add_accessors`.